contribute/architecture/backend/services.md

```diff
@@ -107,31 +107,31 @@ func InitializeForTest(cla setting.CommandLineArgs, opts Options, apiOpts api.Se
 
 ## Background services
 
-A background service is a service that runs in the background of the lifecycle between Grafana starts up and shutdown. If you want a service to be run in the background your Service should satisfy the `registry.BackgroundService` interface and add it as argument to the [ProvideBackgroundServiceRegistry](/pkg/server/backgroundsvcs/background_services.go) function and add it as argument to `NewBackgroundServiceRegistry` to register it as a background service.
+A background service is a service that runs in the background of the lifecycle between Grafana startup and shutdown. If you want a service to be run in the background, your Service should satisfy the `registry.BackgroundService` interface and add it as argument to the [ProvideBackgroundServiceRegistry](/pkg/server/backgroundsvcs/background_services.go) function and add it as argument to `NewBackgroundServiceRegistry` to register it as a background service.
```

Comment:
I'm not sure about the word "satisfy" --is there another word we can use? I think the last sentence has double info-can you take another look?

Reply:
Fixing this sentence; yep, weirdly worded